### PR TITLE
ci: don't run website deployment on forks

### PR DESCRIPTION
**Pull request by senekor**
# Checklist

If applicable:
- [ ] I have updated `CHANGELOG.md`
- [ ] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have updated the config schema (cli/src/config-schema.json)
- [ ] I have added tests to cover my changes

I have manually tested this change on my own fork confirming that the job doesn't run. If I replace "martinvonz" with "senekor", it runs again (but obviously failing), so it should continue to run on the main repo.

### CODE REVIEW

**Review by martinvonz — APPROVED** [review]
Makes sense.

**Comment by senekor** [comment]
The PR description template says "Merge the PR at will once it's been approved.", but I can't do that. GitHub UI says "Only those with write access to this repository can merge pull requests."

**Comment by martinvonz** [comment]
> The PR description template says "Merge the PR at will once it's been approved.", but I can't do that.

Ah, sorry, I thought you already were a collaborator. I've invited you now.

**Pull request merged**
